Dockerfile

```diff
@@ -49,21 +36,18 @@ RUN \
   apt-get update && \
   apt-get install zfp
 
-# gRPC
+# grpc from webispy ppa
 RUN \
-  cd /root && \
-  git clone --recurse-submodules https://github.com/grpc/grpc && \ 
-  mkdir -p grpc/cmake/build && cd grpc/cmake/build && \
-  cmake -DgRPC_INSTALL=ON -DgRPC_BUILD_TESTS=OFF -DCMAKE_INSTALL_PREFIX=/usr/local -DgRPC_SSL_PROVIDER=package ../.. && \
-  make && make install && \
-  cd /root && rm -rf grpc
+  add-apt-repository ppa:webispy/grpc && \
+  apt-get update && \
+  apt-get install -y libprotobuf-dev protobuf-compiler libgrpc++-dev libgrpc-dev protobuf-compiler-grpc googletest
 
 # Build carta-backend (currently checkouts the confluence/generic-scripting branch)
 RUN \
   apt-get -y install libxml2-dev && \
   git clone https://github.com/CARTAvis/carta-backend.git && \
   cd carta-backend && \
-  git checkout confluence/generic-scripting && \
+  git checkout angus/build_test && \
```

Comment:
How does this work when we merge? Shouldn't we remove it when we eventually merge this into `dev`?

Reply:
I guess the sustainable way is to rather use the host's checkout of carta and map it to a `src` directory or something.